files-backup.php

```diff
@@ -1,19 +1,20 @@
 <?php
-
 /**
+ * @file
  * Backup the files directory of a Backdrop CMS site.
  */
 
 // Check if we already have config array.
 if (!isset($config)) {
- $config = parse_ini_file('config.ini');
+  $config = parse_ini_file('config.ini');
 }
 
 // Get path to files directory.
```

Comment:
Can we update or remove this inline comment? It seems not to match the code that follows anymore.

Reply:
Removed.